pom.xml

```diff
@@ -25,10 +25,22 @@
         </dependency>
 
         <dependency>
-            <groupId>commons-validator</groupId>
-            <artifactId>commons-validator</artifactId>
-            <version>${commons-validator.version}</version>
-        </dependency>
+	    <groupId>commons-validator</groupId>
+	    <artifactId>commons-validator</artifactId>
+	    <version>${commons-validator.version}</version>
+	    <exclusions>
+		<exclusion>
+		    <groupId>commons-beanutils</groupId>
+		    <artifactId>commons-beanutils</artifactId>
+		</exclusion>
+	    </exclusions>
+	</dependency>
+
+	<dependency>
+	    <groupId>commons-beanutils</groupId>
+	    <artifactId>commons-beanutils</artifactId>
+	    <version>1.9.4</version>
```

Comment:
Can we have this as a property i.e ${commons.beanutils.version}

Reply:
Yes, I have updated.